spectrum.js

```diff
@@ -9,8 +9,17 @@
     if (typeof define === 'function' && define.amd) { // AMD
         define(['jquery'], factory);
     }
-    else if (typeof exports == "object" && typeof module == "object") { // CommonJS
-        module.exports = factory;
```

Comment:
Would it work if we instead modified this line to be `module.exports = factory(require('jquery'));` and not have the added lines?